src/rules/noUnnecessaryQualifierRule.ts

```diff
@@ -152,3 +152,9 @@ function tryGetAliasedSymbol(symbol: ts.Symbol, checker: ts.TypeChecker): ts.Sym
         ? checker.getAliasedSymbol(symbol)
         : undefined;
 }
+
+/** Wrapper for compatability with typescript@<3.8.2 */
+export function isPrivateIdentifier(node: ts.Node): node is ts.PrivateIdentifier {
```

Comment:
this works, but if you remove the `export`, the rule will be truly backwards compatible (the reference to `ts.PrivateIdentifier` will be elided from typings)

```suggestion
function isPrivateIdentifier(node: ts.Node): node is ts.PrivateIdentifier {
```